pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -77,7 +79,25 @@ This procedure describes how to create a key/value pair secret called `my-secret
    dcos security secrets create --value=top-secret developer/my-secret
    ```
 
-# Creating secrets from a file via the DC/OS Enterprise CLI
+# Creating secrets from any file via the DC/OS Enterprise CLI
+
+This procedure describes how to use any file to create a secret called `myapp.keystore` inside the `developer` path using the DC/OS Enterprise CLI.
+
+The contents of the file (referred to below as `myapp.keystore`) can be any value, including binary and text values. The secrets service encodes binary files into `BASE64` text form before storing it in secret store.
```

Comment:
Choose/use a consistent capitalisation for "secret store" - we also have "Secret Store".

Reply:
I guess "the secret store" or "a secret store".

Reply:
"before storing **them**"